<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1364</width>
    <height>835</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#header {
	background-color: #004d99;
	padding-left: 10px;
	margin-left: 10px;
	margin-right: 10px;
}


#title {
	color: white;
	font-weight: bold;
	font-size: 36px;
}

#btnBackMainMenu {
	background-color: #4db8ff;
	border-radius: 20px;
	color: white;
	margin-top: 10px;
	font-size: 24px;
	height: 50px;
}

#btnBackMainMenu:hover {
	background-color: #1AA4FF;
}

#mainFrame {
	border: 2px solid #0073e6;
	margin-left: 3px;
	margin-right: 10px;
}

#mainFrame #btnPrint {
	color: #ffffff;
	background-color:  #004d99;
	border-radius: 15px;
	font-weight: bold;
	font-size: 24px;
	height: 80px;
	padding: 10px 16px;
	margin-bottom: 10px;
}

#mainFrame #btnPrint:hover {
	background-color: #0067CC;
}

#mainFrame QPushButton {
	margin-top: 10px;
	margin-bottom: 10px;
}

#btnCapture {
	color: #ffffff;
	background-color:  #004d99;
	font-weight: bold;
}

#btnCapture:hover {
	background-color: #2215FB;
}

#btnCheck {
	color: #ffffff;
	background-color: #DDDDDD;
	font-weight: bold;
}

#btnCheck:hover {
	background-color: #00FF00;
}

#btnProfile {
	color: #ffffff;
	background-color:  #DDDDDD;
	font-weight: bold;
}

#btnProfile:hover {
	background-color: #E4843F;
}

#labelItem1, #labelItem2, #labelItem3, #labelItem4, #labelItem5, #labelItem6, #labelItem7, #labelItem8, #labelItem9, #labelItem10,#labelItem11, #labelItem12, #labelItem13, #labelItem14, #labelItem15, #labelItem16, #labelItem17, #labelItem18, #labelItem19, #labelItem20 {
	font-size: 18px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0" colspan="2">
     <widget class="QFrame" name="header">
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="2">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="btnBackMainMenu">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>330</x>
            <y>10</y>
            <width>261</width>
            <height>61</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Back to Main Menu</string>
          </property>
          <property name="icon">
           <iconset resource="../qrc/main_menu_rc.qrc">
            <normaloff>:/images/assets/arrow-left.svg</normaloff>:/images/assets/arrow-left.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
         </widget>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="logo">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../qrc/main_menu_rc.qrc">:/images/assets/logo-white.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="title">
         <property name="text">
          <string>PROJECT: WattWise</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QFrame" name="mainFrame">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1" colspan="2">
        <widget class="QFrame" name="cameraFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>630</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout"/>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QFrame" name="answersFrame_2">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>18</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>YOUR ANSWERS</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="answersFrame">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QFrame" name="firstFrame">
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QLabel" name="labelItem1">
                  <property name="text">
                   <string>1.</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelItem2">
                  <property name="text">
                   <string>2.</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelItem3">
                  <property name="text">
                   <string>3.</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelItem4">
                  <property name="text">
                   <string>4.</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelItem5">
                  <property name="text">
                   <string>5.</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelItem6">
                  <property name="text">
                   <string>6.</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelItem7">
                  <property name="text">
                   <string>7.</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelItem8">
                  <property name="text">
                   <string>8.</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelItem9">
                  <property name="text">
                   <string>9.</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelItem10">
                  <property name="text">
                   <string>10.</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="secondFrame">
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QLabel" name="labelItem11">
                  <property name="text">
                   <string>11.</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelItem12">
                  <property name="text">
                   <string>12.</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelItem13">
                  <property name="text">
                   <string>13.</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelItem14">
                  <property name="text">
                   <string>14.</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelItem15">
                  <property name="text">
                   <string>15.</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelItem16">
                  <property name="text">
                   <string>16.</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelItem17">
                  <property name="text">
                   <string>17.</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelItem18">
                  <property name="text">
                   <string>18.</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelItem19">
                  <property name="text">
                   <string>19.</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelItem20">
                  <property name="text">
                   <string>20.</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QFrame" name="rightFrame">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QFrame" name="optionFrame">
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QPushButton" name="btnCapture">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>CAPTURE</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnCheck">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>CHECK</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnProfile">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>PROFILE</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="resultFrame">
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QGroupBox" name="groupBox">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="title">
                <string>RESULT</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <item>
                 <widget class="QLabel" name="labelResults">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>26</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignHCenter|Qt::AlignTop</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../qrc/main_menu_rc.qrc"/>
 </resources>
 <connections/>
</ui>
